src/index.html

```diff
@@ -1,14 +1,31 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Stop watch</title>
-  <link rel="stylesheet" href="./styles/main.scss">
-</head>
-<body>
-  <h1>Stop watch</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width,
+      user-scalable=no, initial-scale=1.0,
+      maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta
+      http-equiv="X-UA-Compatible"
+      content="ie=edge"
+    >
+    <title>Stop watch</title>
+    <link
+      rel="stylesheet"
+      href="./styles/main.scss"
+    >
+  </head>
+  <body>
+    <main class="page">
+      <section class="watch">
+        <div class="watch__big-circle">
+          <div class="watch__small-circle"></div>
+          <div class="watch__minutes"></div>
+          <div class="watch__seconds"></div>
+        </div>
+      </section>
```

Comment:
No need to create extra wrappers. You can write styles from a class 'watch__big-circle' to class "watch" and delete 1 div.
And check p.1 of the checklist (Don't forget about semantics, div development is not a better option).